diff --git a/tests/daostorm_3d/test_fit.py b/tests/daostorm_3d/test_fit.py
index e086194..da366a2 100644
--- a/tests/daostorm_3d/test_fit.py
+++ b/tests/daostorm_3d/test_fit.py
@@ -220,8 +220,8 @@ class FitterTest(unittest.TestCase):
         f = fit_impl.FitterZ(self.z_sim_img, self.z_sim_local_max,
                              self.z_params, 1e-6)
         f.iterate()
-        np.testing.assert_allclose(f.peaks, orig["peaks"])
-        np.testing.assert_allclose(f.residual, orig["residual"])
+        np.testing.assert_allclose(f.peaks, orig["peaks"], atol=1e-6)
+        np.testing.assert_allclose(f.residual, orig["residual"], atol=1e-6)
 
     def test_fit_z_sim(self):
         # Produced by a test run of this implementation. Differs from
